itests/integration_test.go

```diff
@@ -32,19 +33,19 @@ func (suite *ItestSuite) SetupSuite() {
 	suite.NoError(err, "couldn't create config")
 	suite.cfg = cfg
 
-	docker, err := d.NewDocker()
+	suiteName := strings.ToLower(suite.T().Name())
```

Comment:
Why `ToLower` is necessary here?

Reply:
I think `ToLower` makes every string better

Reply:
I think the reason for `ToLower` here is different :)

Reply:
It necessary because docker doesn't work with uppercase in the container name